src/app/components/menu/menu.component.scss

```diff
@@ -17,6 +17,7 @@ option{
 
 .genres div, .select-sort {
   padding: 20px;
+  z-index: 10;
```

Comment:
it's not a good practice to have randomly set z-index for elements, for medium and big sizes projects, people usually have z-index in one place with a good variable name so for supporters it'll be easier to debug or add new layers